app/ui/legacy/src/main/java/com/fsck/k9/activity/AlternateRecipientAdapter.java

```diff
@@ -167,6 +167,11 @@ public void onClick(View v) {
 
         holder.copyEmailAddress.setOnClickListener(v -> listener.onRecipientAddressCopy(currentRecipient));
 
+        holder.cutEmailAddress.setOnClickListener(v -> {
```

Comment:
Implicit lambda can't be removed here:)

Reply:
I've recently checked the code, we're declaring the receiver of lambda everywhere in the code. I guess, You can skip this. Thanks.

Reply:
Oh, you mean the `v ->` to `_ ->`? If you think is best, I can apply that change 👍